libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -269,6 +269,19 @@ func (m *Manager) Apply(pid int) error {
 			newProp("CPUShares", uint64(c.Resources.CpuShares)))
 	}
 
+	// cpu.cfs_quota_us and cpu.cfs_period_us are controlled by systemd.
+	// The percentage cpuQuota refered specifies how much CPU time the unit
+	// shall get at maximum, relative to the total CPU time available on one CPU.
+	// The percentage should be greater than 1%.
+	if c.Resources.CpuQuota != 0 && c.Resources.CpuPeriod != 0 {
+		cpuQuota := c.Resources.CpuQuota * 100 / c.Resources.CpuPeriod
+		if cpuQuota == 0 {
+			return fmt.Errorf("cpu_quota divided by cpu_period must be greater than 0.01")
```

Comment:
Can you provide a link to the systemd documentation which describes this option? I can't find it [here](https://www.freedesktop.org/software/systemd/man/systemd.resource-control.html). If it's an option added recently, you'll need to make your change backwards compatible with older systemd versions (by conditionally adding it like we do with `Delegate` for example).